src/LoopDriver.php

```diff
@@ -40,7 +51,9 @@ public function defer(callable $callback, $data = null);
     public function delay(callable $callback, $delay, $data = null);
 
     /**
-     * Repeatedly execute a callback. The interval between executions is approximate and accuracy is not guaranteed.
+     * Repeatedly execute a callback.
+     *
+     * The interval between executions is approximate and accuracy is not guaranteed.
```

Comment:
Perhaps specify a bit more precisely that the time between two callbacks is _at least_ $interval. This should be guaranteed. Same for delay().